mitodl/ocw-studio
# Collaborator add/edit modal drawer

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #166 

#### What's this PR do?
Moves the add/edit contributor forms to a modal drawer

#### How should this be manually tested?
Go to the collaborators page.  Add, edit, and delete collaborators.  All 3 should work as expected without errors.

#### Screenshots (if appropriate)

<img width="1194" alt="Screen Shot 2021-04-05 at 12 20 00 PM" src="https://user-images.githubusercontent.com/187676/113601835-dceb0e00-960f-11eb-9581-bfd942d0ae5e.png">

<img width="1189" alt="Screen Shot 2021-04-05 at 12 20 12 PM" src="https://user-images.githubusercontent.com/187676/113601846-dfe5fe80-960f-11eb-8d55-fdccdc25da58.png">
